addons/grenades/CfgAmmo.hpp

```diff
@@ -103,6 +103,7 @@ class CfgAmmo {
 
     class ACE_G_M84: SmokeShell {
         GVAR(flashbang) = 1;
+
```

Comment:
Was this added on purpose? Is it to ensure that the file is marked as changed by this commit?

Reply:
Didn't delete the newline after debugging. I usually put a space between vanilla and addon config entries.